README.md

```diff
@@ -10,6 +10,10 @@ This plugin provides dependency metadata for Python projects that use one of the
 
 * `pip` with a `requirements.txt` file
 * `pipenv` with a `Pipefile` file
+* `setup.py` file (setuptools/distutils)
```

Comment:
Please stay away about mentioning this support, it hasn't been fully tested and propagated yet.